xml/System/String.xml

```diff
@@ -5301,14 +5301,14 @@ Invalid sequences are represented in the enumeration by <xref:System.Text.Rune.R
 
  In this section:
 
- [Get started with the String.Format method](#Starting)
- [Which method do I call?](#FTaskList)
- [The Format method in brief](#Format_Brief)
- [The Format item](#FormatItem)
- [How arguments are formatted](#HowFormatted)
- [Format items that have the same index](#SameIndex)
- [Formatting and culture](#Format_Culture)
- [Custom formatting operations](#Format_Custom)
+ [Get started with the String.Format method](#Starting)\
```

Comment:
In the conceptual docs, we've standardized on lists for these links. I'll approve, and let you consider which you prefer.